addons/website_forum/models/forum_post.py

```diff
@@ -51,7 +51,8 @@ class Post(models.Model):
     bump_date = fields.Datetime('Bumped on', readonly=True,
                                 help="Technical field allowing to bump a question. Writing on this field will trigger "
                                      "a write on write_date and therefore bump the post. Directly writing on write_date "
-                                     "is currently not supported and this field is a workaround.")
+                                     "is currently not supported and this field is a workaround.",
+                                required=True, default= fields.Datetime.now())
```

Comment:
I would update this as we are using this bump date now instead of write_date. It is no longer a hack, and we do this to catch new replies/comments and not "favoriting".